README.md

```diff
@@ -7,6 +7,17 @@ A python implementation of the UA Parser (https://github.com/ua-parser, formerly
 
 ##Installing
 
+### Install via pip
+First, check that you've got the correct dependencies. For example, on Ubuntu:
+```
+$ apt-get install libyaml libyaml-dev python-dev
```

Comment:
Yep, but I can confirm that imports will fail if you install via pip without prior installing these (see https://github.com/ua-parser/uap-python/issues/14#issuecomment-135792816)